images/ceph/Makefile

```diff
@@ -22,7 +22,7 @@ CEPH_VERSION = v15.2.9-20210224
 else
 CEPH_VERSION = v15.2.9-20210224
 endif
-BASEIMAGE = ceph/ceph-$(GOARCH):$(CEPH_VERSION)
+BASEIMAGE = ceph/daemon-base:latest-master-devel # ceph/ceph-$(GOARCH):$(CEPH_VERSION)
```

Comment:
We don't actually need Pacific in the operator image, right? I don't anticipate updating the base image until 1.7 unless we really need to sooner.

Reply:
I'm not sure. I thought we did in order to use the new `--max` flag. I guess I should test that.

Reply:
Wow, you're right! I'm really surprised the Ceph CLI is forward compatible like that.